packages/stateManager/tests/mockClient.ts

```diff
@@ -61,7 +61,11 @@ export class MockClient implements IRelayClient {
       resolve;
     });
   }
-
+  authenticate() {
+    return new Promise((resolve) => {
+      resolve;
```

Comment:
just the mock but shouldn't resolve be called here? cc @wanderer

Reply:
`return Promise.resolve()` should work